misc/deployment/BuyerWeb/helm-chart/templates/deployment.yaml

```diff
@@ -0,0 +1,83 @@
+apiVersion: apps/v1
```

Comment:
**suggestion:** In the deployment.yaml files there are missing 'buyer-web-react-ssr' missing. Upon deployments both of the services the buyer-web and buyer-web-react be deployed together because it's one app